plugins/filter/to_prettytable.py

```diff
@@ -0,0 +1,180 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2025, Timur Gadiev <tgadiev@gmail.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import (absolute_import, division, print_function)
+
+DOCUMENTATION = '''
+  name: to_prettytable
+  short_description: Format a list of dictionaries as an ASCII table
+  version_added: "10.6.0"
+  author: Timur Gadiev (@tgadiev)
+  description:
+    - This filter takes a list of dictionaries and formats it as an ASCII table using the I(prettytable) Python library.
+  requirements:
+    - prettytable
+  options:
+    _input:
+      description: A list of dictionaries to format.
+      type: list
+      elements: dictionary
+      required: true
+    column_order:
+      description: List of column names to specify the order of columns in the table.
+      type: list
+      elements: string
+    header_names:
+      description: List of custom header names to use instead of dictionary keys.
+      type: list
+      elements: string
+    column_alignments:
+      description: >-
+        Dictionary where keys are column names and values are alignment settings.
+        Valid alignment values are C(left), C(center), C(right), C(l), C(c), or C(r).
+        For example, {'name': 'left', 'id': 'right'} will align the 'name' column to the left
+        and the 'id' column to the right.
```

Comment:
1. I'd split this up into two paragraphs. That makes it much more readable.
2. Use formatting for the example.
```suggestion
      description:
        - >-
          Dictionary where keys are column names and values are alignment settings.
          Valid alignment values are C(left), C(center), C(right), C(l), C(c), or C(r).
        - >-
          For example, V({'name': 'left', 'id': 'right'}) will align the C(name) column to the left
          and the C(id) column to the right.
```

Reply:
Corrected as requested.